INTERPRETING.md

```diff
@@ -135,9 +135,11 @@ This must precede any additional text modifications described by test metadata.
 ### Modules
 
 Test262 includes tests for ECMAScript 2015 module code, denoted by the "module"
-metadata flag. Files bearing a name ending in `_FIXTURE.js` should not be
-interpreted as standalone tests; they are intended to be referenced by test
-files.
+metadata flag. Files bearing a name ending in `_FIXTURE.js`:
+
+- **MUST NOT** be interpreted as standalone tests; they are intended to be referenced by test files.
+- **MUST NOT** refer to, or make use of any [Test262-Defined Bindings](#test262-defined-bindings) in any way. 
+- **MUST NOT** refer to, or make use of any [Host-Defined Functions](#host-defined-functions) in any way. 
```

Comment:
@rwaldron Come to think of it, the last two bullet points are actually contribution guidelines. For this document, we might want to instead extend the instruction about "must not be interpreted as standalone tests" to say how the Realm modifications described elsewhere in this document must not be applied.

Sorry to be getting to this so late